packages/core/src/components/icon/icon.stories.ts

```diff
@@ -9,7 +9,12 @@ export default {
   title: 'Core/Icon',
   component: Icon,
   argTypes: {
-    color: { control: { type: 'select', options: colors } },
+    color: {
+      control: { type: 'select', options: colors },
+      table: {
+        type: { summary: colors.map((value) => `"${value}"`).join('|') },
```

Comment:
💅🏻  We could set this function as `colors.toString` so that we can reuse it across files

Reply:
Always commenting after merge 😬 

Don't worry about it

Reply:
Feel free to do a PR with this! 😏